apps/src/applab/applab.js

```diff
@@ -782,7 +776,8 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugSlider: showDebugConsole,
-    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers')
+    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
+    debuggerUi,
```

Comment:
All of our other pageConstants are (I believe) simple bools or strings. This sort of seems like it should be a separate concept.

It actually feels a little weird to me to have this be in our redux store in general. It isn't really state, right? It's a bunch of code.